src/main/java/io/jawg/osmcontributor/flickr/oauth/OAuthRequest.java

```diff
@@ -58,7 +58,7 @@ public OAuthRequest(String apiKey, String apiKeySecret) {
     /*----------------CODE---------------------*/
     /*=========================================*/
     public void initParam(Map<String, String> params) {
-        this.params.clear();
+        this.params = new TreeMap<>();
```

Comment:
Hadn't seen that there was a TreeMap used. Please use HashMap instead (better efficiency in read / write as it is not sorted)